.vscode/cspell.json

```diff
@@ -0,0 +1,63 @@
+{
+  "version": "0.1",
+  "language": "en",
+  "languageId": "java",
+  "dictionaries": ["powershell"],
+  "ignorePaths": [
+    "**/session-records/**",
+    "*.jar",
```

Comment:
Why is this list extra long compared to the other repo's?

Reply:
I ran cspell scans of whole repos at the beginning of this project. cspell scans all files and does not ignore file extensions commonly associated with binary data (e.g. .png, etc.) where there is no easy way to determine what collection of adjacent bytes make up words to spell check. After the scan completed (many hours of execution time) I combed through the results to eliminate file extensions. In some cases ignoring these files improved the performance of the scanning significantly.